packages/node_modules/@ciscospark/plugin-messages/src/messages.js

```diff
@@ -28,6 +42,71 @@ import {isArray} from 'lodash';
  * @class
  */
 const Messages = SparkPlugin.extend({
+  /**
+   * Initializer
+   * @param  {...any} args
+   * @returns {undefined}
+   */
+  initialize(...args) {
+    Reflect.apply(SparkPlugin.prototype.initialize, this, args);
+  },
+
+  /**
+   * Connect to the web socket to listen to incoming messages.
+   * @returns {Promise}
+   */
+  listen() {
+    return this.spark.internal.mercury.connect()
+      .then(() => this.listenTo(this.spark.internal.mercury, INCOMING_MESSAGES_CREATED,
+        (event) => this.onConversationActivityEvent(event)));
+  },
+
+  /**
+   * Trigger a "created" event.
+   * @param {Object} event
+   * @returns {undefined}
+   */
+  async onConversationActivityEvent(event) {
```

Comment:
I don't see where this needs to be async

Reply:
It doesn't have to anymore. I had it fetching something else at one point. Good catch!